openfeature/api.py

```diff
@@ -10,15 +10,13 @@
 from openfeature.exception import GeneralError
 from openfeature.hook import Hook
 from openfeature.provider import FeatureProvider
+from openfeature.provider._registry import default_registry
```

Comment:
Could we call this provider_registery? I don't find default_registery that intuitive unless we add a nondefault_registery.

Reply:
Great point! I was also feeling a bit awkward about the name.